### ISSUE TITLE
Use joomla tags to have a module for similar articles

### ISSUE THREAD

**Issue by houbahoubi** [GavickPro/News-Show-Pro-GK5#278]
hello

On the advice of presale department, I contact you to know if we can use this module with the new joomla tags functionnality in J3. 
Because the default joomla similar tags module shows similar articles due to tags in articles. That's fine but "ugly" : not introtext, no intro image, only article title...
So is there a way to do the same with your module ? I don't see it in the parameters.
Thanks

**Comment by dziudek** [comment]
Hello,

In fact you can implement this by selecting as a data source the joomla tags, unfortunately for every article you will have to create a separate instance with different tags.

**Comment by houbahoubi** [comment]
Hello
Thanks for your answer. 
The problem is that there is no relation with the current article, so if I well understand I need to create a module for each article ! The use is for a blog so it is not possible.
Thanks

**Comment by dziudek** [comment]
Yes, exactly - it will be very problematic. I will leave this issue open and I will try to create a similar tags data source in one of the next module versions.

**Comment by houbahoubi** [comment]
ok thanks for your help. I hope you can do it in a future version ;-)

**Comment by dziudek** [comment]
As I wrote earlier - I will leave this issue reopen ;)
